.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This is config for Prettier, a tool that automatically adds formatting and spacing to the code. Prettier has a lot of opinionated defaults, and this file says what should be different. (docs: [Prettier configuration file](https://prettier.io/docs/en/configuration.html))